packages/taquito/src/taquito.ts

```diff
@@ -348,6 +350,13 @@ export class TezosToolkit {
     return this._context.contract;
   }
 
+  /**
+   * @description Provide access to tezos operation preparation utilities
+   */
+  get prepare(): PreparationProvider {
+    return this._context.prepare;
```

Comment:
I would use lambda syntax for all 1-line methods but that's just a stylistic preference. In general I like to see as much code as possible, with spaces only between method declarations, most of the time (and: assumes small, focused methods),